### DISCUSSION TITLE
Setting up GCS IO Manager by default to all the jobs

### DISCUSSION THREAD

**Discussion by kennycontreras** [dagster-io/dagster#18420] [Q&A] [answered]
I'm configuring IO Manager for our Dagster environment, and I'd like to know if there's a way to set GCS IO Manager by default to all the jobs instead of declaring the IO Manager for each job created in the workspace. 

I'm following this [guide](https://docs.dagster.io/deployment/guides/gcp#using-gcs-for-io-management
), but I wonder if this will apply to the rest of the jobs. 

Thanks!

**Comment by yuhan** [comment, marked as answer]
I/O managers are resources. By default, resources defined on `Definitions` are automatically bound to jobs. So you can configure your I/O managers on `Definitions`. Here's an example: https://docs.dagster.io/concepts/io-management/io-managers#applying-io-managers-to-assets
